docs/sharded-computation.md

```diff
@@ -12,17 +12,19 @@ kernelspec:
 ---
 
 (sharded-computation)=
-# Introduction to sharded computation
+# Introduction to sharded computation in JAX
 
-JAX's {class}`jax.Array` object is designed with distributed data and computation in mind.
+This tutorial serves as an introduction to device parallelism for Single-Program Multi-Data (SPMD) code in JAX. SPMD is a parallelism technique where the same computation, such as the forward pass of a neural network, can be run on different input data (for example, different inputs in a batch) in parallel on different devices, such as several GPUs or Google TPUs.
 
-This section will cover three modes of parallel computation:
+The tutorial covers three modes of parallel computation:
 
-- Automatic parallelism via {func}`jax.jit`, in which we let the compiler choose the optimal computation strategy
-- Semi-automatic parallelism using {func}`jax.jit` and {func}`jax.lax.with_sharding_constraint`
-- Fully manual parallelism using {func}`jax.experimental.shard_map.shard_map`
+- _Automatic parallelism via {func}`jax.jit`_: The compiler chooses the optimal computation strategy (a.k.a. "the compiler takes the wheel"). Based on the sharding of the data, the compiler automatically partitions bulk array functions over multiple devices, dividing shardings for intermediate and output values, and parallelizes their evaluation, even inserting communication operations when necessary.
```

Comment:
I think we should remove the second sentence. It's too much detail for a bullet-point summary of what's to come.

Reply:
For advanced users, move it to a separate paragraph: 

"**Note:** With {func}`jax.jit`, based on the sharding of the data, the compiler automatically partitions bulk array functions over multiple devices, dividing shardings for intermediate and output values, and parallelizes their evaluation, even inserting communication operations when necessary."